l5kit/l5kit/rasterization/box_rasterizer.py

```diff
@@ -49,9 +49,9 @@ def draw_boxes(
         np.ndarray: the image with agents rendered. RGB if color RGB, otherwise GRAY
     """
     if isinstance(color, int):
-        im = np.zeros(raster_size, dtype=np.uint8)
+        im = np.zeros((raster_size[1], raster_size[0]), dtype=np.uint8)
```

Comment:
can we avoid swapping box_rasterizer width and height? This is counter-intuitive

Reply:
Our codebase is XY, but numpy wants shape as YX so I don't really know how we can avoid that, any idea?

Reply:
okay I see. Yeah, I do not have good suggestion